internal/info/info_test.go

```diff
@@ -392,76 +393,270 @@ func TestDetail_prepare(t *testing.T) {
 		NGTVersion        string
 		BuildCPUInfoFlags []string
 		StackTrace        []StackTrace
-		PrepOnce          sync.Once
 	}
 	type want struct {
+		want *Detail
 	}
 	type test struct {
 		name       string
 		fields     fields
 		want       want
-		checkFunc  func(want) error
+		checkFunc  func(want, *Detail) error
 		beforeFunc func()
 		afterFunc  func()
 	}
-	defaultCheckFunc := func(w want) error {
+	defaultCheckFunc := func(w want, got *Detail) error {
+		if w.want.GitCommit != got.GitCommit {
```

Comment:
please consider using go-cmp
https://godoc.org/github.com/google/go-cmp/cmp
I have used it here
https://github.com/vdaas/vald/pull/501/files#diff-8194d570432526219ea1cec4d8b45e6aR88

Reply:
thank you for your review!
I will use it.